src/ngChosen.ts

```diff
@@ -79,7 +80,15 @@ module ngChosen {
 			} else {
 				this.updateState(elem, false, false, false);
 			}
-
+            
+            if (scope.ngModel !== undefined) {
+                scope.$watch("ngModel", function(newValue, oldValue) {
```

Comment:
I would check if both elements are different before doing the other checks.

Also indentation seems a little weird in all the lines you've added